src/core/js/views/notifyView.js

```diff
@@ -60,13 +66,19 @@ define([
         },
 
         render: function() {
-            var data = this.model.toJSON();
-            var template = Handlebars.templates['notify'];
+            var data, html;
```

Comment:
Out Style-Guide recommends to [declare one variable per var statement](https://github.com/adaptlearning/documentation/blob/master/01_cross_workstream/style_guide.md#declare-one-variable-per-var-statement).